### PR TITLE
#426 adding a message if invited and accepted is false

### LINKED ISSUE #426: Make sure "invited and accepted" header is populating on Users index
In prod this column is completely empty. Please check that invited && accepted is displaying correctly and fix if necessary.

### PR DESCRIPTION
**Pull request by renatamarques97**
here is my proposal for resolves #426

# Checklist:
- I have performed a self-review of my own code,
- I have added tests that prove my fix is effective or that my feature works,
- New and existing unit tests pass locally with my changes ("bundle exec rake")
ps: there is a spec failing, but it's not related to my PR.

### Screenshots
Screen Shot 2020-03-22 at 14 17 51

### CODE REVIEW

File: app/views/user_management/show.html.erb [hunk 1]
```diff
@@ -8,8 +8,12 @@
...
       <td> <%= user.email %> </td>
       <td>
-        <% if user.invitation_sent_at? %>
-          <%= user.invitation_accepted? %>
+        <% if user.invitation_sent_at? && user.invitation_accepted? %>
+          <%= "Invited and Accepted" %>
+        <% elsif user.invitation_sent_at? || user.invitation_accepted? %>
+          <%= "Invited but not Accepted yet" %>
+        <% else %>
+          <%= "Not Invited" %>
```
Comment (sobrinho) [inline app/views/user_management/show.html.erb]: Better:

```erb 
<% if user.invitation_accepted? %>
  Invited and accepted
<% elsif user.invitation_sent_at? %>
  Invited but not accepted yet
<% else %>
  Not invited
<% end %>
```

File: spec/views/user_management/show.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "user_management/show.html.erb", type: :view do
+  before(:each) do
+    @user_not_invited = create(:user)
+  end
+
+  it 'renders columns' do
```
Comment (sobrinho) [inline spec/views/user_management/show.html.erb_spec.rb]: If you want to test what has been added, you must test all the three cases.

**Review by sobrinho — CHANGES_REQUESTED** [review]
See comments.

File: spec/views/user_management/show.html.erb_spec.rb (hunk 2 above)
Comment (renatamarques97) [reply spec/views/user_management/show.html.erb_spec.rb]: I know, but I still don't know how to do it, could you help me?

File: app/views/user_management/show.html.erb (hunk 1 above)
Comment (renatamarques97) [reply app/views/user_management/show.html.erb]: thanks :D

File: spec/views/user_management/show.html.erb_spec.rb (hunk 2 above)
Comment (sobrinho) [reply spec/views/user_management/show.html.erb_spec.rb]: Sure, I'm going to ping you pvt.

**Review by sobrinho — APPROVED** [review]
Nice work!

**Comment by maebeale** [comment]
@renatamarques97 thank you! have you pulled the latest changes from develop? circleci should pass at this point.

**Comment by renatamarques97** [comment]
@maebeale okay.

**Comment by renatamarques97** [comment]
@maebeale you sure it should pass?
Screen Shot 2020-03-24 at 12 07 08

**Comment by sobrinho** [comment]
Yep, it's not passing: https://github.com/rubyforgood/voices-of-consent/commit/d0cbcfacdde73e59149d4bdd6dc6cad9688d4f19

**Comment by sobrinho** [comment]
@maebeale see #448 for fixing specs.

**Pull request merged**